modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java

```diff
@@ -322,6 +357,20 @@ public void run() {
     }
 
     @ReactMethod
+    public void stopNode() {
+
+        Thread thread = new Thread() {
+            @Override
+            public void run() {
+                Log.d(TAG, "stopNode");
```

Comment:
StopNode if we want to be consistent

Reply:
ok

Reply:
Actually no, we are consistent here because that's the name of Java method.

Reply:
ok